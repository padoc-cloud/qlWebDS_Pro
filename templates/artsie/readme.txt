Theme Name: Artsie
Theme URI: http://www.categorydirectory.com
Description: Sponsored CSS/XHTML Template for qlWebDS Pro v. 6.3.*
Version: 6.0
Author: CategoryDirectory.com
Author URI: http://www.categorydirectory.com
