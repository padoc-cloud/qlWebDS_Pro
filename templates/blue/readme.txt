Theme Name: Blue
Theme URI: http://www.directorycheck.com
Description: Free CSS Template for qlWebDS Pro v. 6.3.*
Version: 6.0
Author: DirectoryCheck.com
Author URI: http://www.directorycheck.com
