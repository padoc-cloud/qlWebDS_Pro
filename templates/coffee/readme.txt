Theme Name: Coffee
Theme URI: http://www.regularlink.com
Description: Free CSS Template for qlWebDS Pro v. 6.3.*
Version: 6.0
Author: RegularLink.com
Author URI: http://www.regularlink.com
