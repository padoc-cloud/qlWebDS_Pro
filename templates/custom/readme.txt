Theme Name: Custom
Theme URI: http://www.qlweb.com
Description: This is a Duplicate of the Free Default CSS Template for qlWebDS Pro v. 6.3.*. Use it to develop new CSS Templates. This way you'll always have a clean copy of the Default one.
Version: 6.0
Author: qlWeb.com
Author URI: http://www.qlweb.com
