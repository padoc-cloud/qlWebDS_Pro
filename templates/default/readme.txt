Theme Name: Default
Theme URI: http://www.qlweb.com
Description: Free Default CSS Template for qlWebDS Pro v. 6.3.*
Version: 6.0
Author: qlWeb.com
Author URI: http://www.qlweb.com
