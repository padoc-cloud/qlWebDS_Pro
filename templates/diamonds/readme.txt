Theme Name: Diamonds
Theme URI: http://www.directorysnob.com
Description: Sponsored CSS/XHTML Template for qlWebDS Pro v. 6.3.*
Version: 6.0
Author: DirectorySnob.com
Author URI: http://www.directorysnob.com
