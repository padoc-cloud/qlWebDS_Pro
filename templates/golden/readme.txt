Theme Name: Golden
Theme URI: http://www.somanylinks.com
Description: Free CSS Template for qlWebDS qlWebDS Pro v. 6.3.*
Version: 6.0
Author: SoManyLinks.com
Author URI: http://www.somanylinks.com
